docs/guides/build/auto-generated-docs.md

```diff
@@ -1,56 +1,48 @@
+The product documentation is generated by parsing the following files in the relevant product directory of your organisation's `proto` repo:
+1. The `resources` and `services` proto files which are used to generate the reference documentation based on the contents of each proto file as per their latest release (`alis proto release`).
+2. The custom markdown files contained within the `docs` folder, used to generate the landing page, guides and samples.
 ```
 proto
 ├──{orgID}
 │   └── {productID}
-│       ├── docs/*.md
-│       ├── resources/*.proto
-│       └── services/*.proto
-└── {productID_2}/
+│       ├── docs/.../*.md
+│       ├── resources/.../*.proto
+│       └── services/.../*.proto
 ```
-From these files, the documentation which is generated is broken up into 2 parts:
-1. Custom Documentation: these are the guides and code samples written by the user and contained within the `docs` folder.
-2. Reference documentation: this is generated directly from the contents of the `.proto` files, as they existed at the latest `alis proto release` command, and is used for documenting the APIs offered by the product.
 
-Once a user is happy with the state of their markdown files for custom documentation, these should be pushed to the master branch of the proto repo as this is the source of truth for generating custom documentation. They should also be satisfied with the state of the currently released protos as these are the source of truth for reference docs. The user can then run `alis docs release {orgID}.{productID}` to publish the documentation for external use.
+Once the documentation has been released, it will be publicly available at [https://{orgID}.{productID}.{orgDomain}]() (eg. [https://docs.de.alis.services](https://docs.de.alis.services).
+
+## Updating reference documentation
+
+The reference documentation component is generated from the proto files. The emphasis on the definition first-approach ensures that the effort is focussed on the design of the protos with detailed comments above the various services, methods, messages and fields and this design is then propagated into the reference documentation for the APIs.
+
+::: danger 
+A current caveat is that the use of certain Markdown syntax in your comments may result in invalid Markdown syntax in the generated documentation, causing the build of your documentation to fail. This is primarily the usage of the “|” character in your proto comments as well as using line breaks in links.
+:::
 
-The documentation will be available at: [https://{orgID}.{productID}.{orgDomain}]()
+## Writing guides and samples
 
-## Writing custom documentation
-To be able to write custom documentation, one must confirm the following
-1. If a docs folder does not exist for the product within the proto repo, create it.
-2. This docs folder must then have, as a minimal starting point, the `guides`, `reference`  and ‘samples’ sub-directories as well as an index.md at each level as indicated below
```

Comment:
@ruanspies the current situation, however, is that there are existing products that do not have a docs folder. Do we account for these in our documentation?